tarodb/sqlc/queries/assets.sql

```diff
@@ -237,6 +237,10 @@ JOIN genesis_info_view
 JOIN key_group_info_view
     ON assets.genesis_id = key_group_info_view.gen_asset_id;
 
+-- name: AllGroups :many
+SELECT *
```

Comment:
I think we can make the query a bit smaller, assuming we only need the plain/tweaked group keys (can just select those).